docker/build/installers/install_intel_ad_rss_lib.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2019 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# Fail on first error.
+set -e
+
+REPO="https://github.com/intel/ad-rss-lib"
+TAG="v1.1.0"
+DIR="/home/tmp/ad-rss-lib"
+
+git clone "${REPO}" "${DIR}"
+pushd "${DIR}"
+git checkout "${TAG}"
+rm -fr .git
```

Comment:
How about downloading the release tar ball directly and then unzip?
https://github.com/intel/ad-rss-lib/archive/v1.1.0.zip

Reply:
It would be ok,i just reference the workspace.in file,see the rss is a git repo. I would close this request,I just hope those two lib could put into docker.